src/pages/ticketholderlist/TicketHolderList.tsx

```diff
@@ -0,0 +1,123 @@
+
+  const handleToggleButton = () => {
+    setDetail((prop) => !prop);
+  };
+  const count = RESPONSE_TICKETHOLDER.data.totalScheduleCount; //나중에 api로 받아와서 반영해야함. state로 바꿀 필요 있을까?
+
+  const filteredData = responseData.filter((obj) => {
+    const isScheduleMatched =
+      schedule === 0 ||
+      (obj.scheduleNumber === "FIRST" && schedule === 1) ||
+      (obj.scheduleNumber === "SECOND" && schedule === 2) ||
+      (obj.scheduleNumber === "THIRD" && schedule === 3);
+    const isPaymentMatched = payment === undefined || obj.isPaymentCompleted === payment;
+
+    return isScheduleMatched && isPaymentMatched;
+  });
+  //도영이가 axios 사용하면 useEffect 필요없다고 했는데, 나중에 리팩토링 할 수도 있음.
+  useEffect(() => {
+    const totalCount = filteredData.reduce(
+      (totalSum, obj) => obj.purchaseTicketCount + totalSum,
+      0
+    );
+    setReservedCount(totalCount);
+    //그리고 여기서 바로 다시 axios 요청 쏘는 로직 구성해두기
+  }, [filteredData]);
+  //이해하기 어려울 것 같아 주석 남깁니다. 모든 회차, 입금 상태 2가지 필터를 사용하여 원하는 결과만 가져오는 형식입니다.
+  //schedule ===0 일 경우는 전체 회차, payment === undefined 일 경우는 전체 입금 여부(입금했든 안했든 렌더링)을 의미합니다.
+
+  return (
+    <>
+      <Banner image={eximg} reservedCount={reservedCount} isOutdated={isOutdated} />
+      <S.BodyWrapper>
+        <S.BodyLayout>
+          <S.LayoutHeaderBox>
+            <S.LayoutFilterBox>
+              {/*set 함수 직접 넘기는 거 안좋다고 했지만, 내부에서 감싸야 하므로 넘김 */}
+              <NarrowDropDown
+                schedule={schedule}
+                payment={payment}
+                totalScheduleCount={count}
+                setSchedule={setSchedule}
+              >
+                모든 회차
+              </NarrowDropDown>
+              <NarrowDropDown
+                schedule={schedule}
+                payment={payment}
+                totalScheduleCount={count}
+                setPayment={setPayment}
+              >
+                입금 상태
+              </NarrowDropDown>
+            </S.LayoutFilterBox>
+            <S.ToggleWrapper>
+              {detail ? (
+                <>
+                  <S.ToggleText>자세히</S.ToggleText>
+                  <S.ToggleOnIcon
+                    onClick={handleToggleButton}
+                    $width={"5.6rem"}
+                    $height={"3.3rem"}
+                  />
+                </>
+              ) : (
+                <>
+                  <S.ToggleText>간략히</S.ToggleText>
+                  <S.ToggleOffIcon
+                    onClick={handleToggleButton}
+                    $width={"5.6rem"}
+                    $height={"3.3rem"}
+                  />
+                </>
+              )}
+            </S.ToggleWrapper>
+          </S.LayoutHeaderBox>
+          {filteredData.map((obj, index) => (
+            <ManagerCard
+              key={`managerCard-${index}`}
+              bookingId={obj.bookingId}
+              isPaid={obj.isPaymentCompleted}
+              isDetail={detail}
+              setPaid={setResponseData}
```

Comment:
p3) setState 함수를 props로 넘기는 것보다 함수를 따로 만들어서 넘기는 게 좋아! 
이거 관련해서 화랑이가 아티클 쓴 게 있어서 링크 남길게! 
https://velog.io/@thisishwarang/React-setState-%EC%9E%90%EC%B2%B4%EB%A5%BC-props%EB%A1%9C-%EC%A0%84%EB%8B%AC%ED%95%98%EC%A7%80-%EB%A7%90%EC%95%84%EC%95%BC-%ED%95%98%EB%8A%94-%EC%9D%B4%EC%9C%A0
```suggestion
// ManagerCard 컴포넌트 내부의 handlePaymentToggle 함수를 밖으로 뺴고, 이 함수를 인자로 넣자!
const handlePaymentToggle = () => {
    setPaid((arr) =>
      arr.map((item) =>
        item.bookingId === bookingId
          ? { ...item, isPaymentCompleted: !item.isPaymentCompleted }
          : item
      )
    );
  };
```

Reply:
좋은 의견입니다! 처음엔 bookingId가 ManagerCard 컴포넌트에만 존재해서, 해당 컴포넌트 안에서 함수를 작성해야 한다고 생각했는데, 생각해보니 애초에 map으로 obj의 속성을 주니까 그때  () => 을 활용해서 함수를 작성할 수 있었더라구요. 덕분에 고민해보고 수정할 수 있었습니다 👍